README.md

```diff
@@ -1,2 +1,66 @@
 # ChapPress
-A simple project to model best practices for WordPress development.
+ChapPress is a simple WordPress project initiated by the Web and Interactive Marketing team for Chapman University. It is intended to to model best practices for WordPress development. This repository will host the Wordpress model site with custom child themes and plugins.
+
+## Goals
+This WordPress project is intended to provide:
+- A consistent local development environment
+- Source control using standard Git workflow practices
+- Modern automated tests
+- Documentation for best practices and standards
+
+## Requirements
+- Homebrew (1.2.3)
+  - MariaDB (10.0 or greater)
+  - Wordpress Command Line (1.2.1)
+- PHP 7
+- Xcode (8.3.3)
+
+## Installation
+
+  Installation process for the setting up Wordpress on a local environment.
+
+  ### Create the MySQL database using the MariaDB monitor.
+
+  ```
+  mysql -uroot
+  CREATE DATABASE databasename;
+  GRANT ALL PRIVILEGES ON databasename.* TO "wordpressusername"@"localhost" IDENTIFIED BY "password";
+  FLUSH PRIVILEGES;
+  EXIT
+  ```
+
+  ### Download WordPress and run the server.
+
+  ```
+  cd /public/
+  wp core download
+  php -S localhost:8000
```

Comment:
The public directory (with the WordPress core code) should be in the repository. You'll need to add it.

You'll also want to make sure no sensitive stuff is committed to the repository like config file with passwords. If you have questions about how to handle this, ping me.

 So I'd expect install instructions at this step to be something like:

```
git clone git@github.com:chapmanu/chap-press.git
cd chap-press/public
cp -v wp-config.php{-dist,}

# Update db settings in wp-config.php

php -S localhost:8000
```